src/ralph/lib/external_services/base.py

```diff
@@ -50,3 +42,14 @@ def run(self, **kwargs):
         while job and not any([job.is_finished, job.is_failed]):
             time.sleep(0.1)
         return job.result
+
+    def run_async(self, **kwargs):
+        job = self.queue.enqueue(self.method, kwargs=kwargs)
+        return job
+
+
+class InternalService(ExternalService):
```

Comment:
we should discuss if this is proper place for it (and if it should stay named `InternalService`)